chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -887,7 +887,7 @@ def zfs_devices_exist(cls):
         return any(lustre_device['backend_filesystem'] == 'zfs' for lustre_device in config['lustre_devices'])
 
     def create_zpools(self):
-        xs = config['lustre_servers']
+        xs = config['lustre_servers'][:4]
```

Comment:
This is starting to get very specific to our particular test clusters.

While I can understand the desire to get tests running reliably on those particular test clusters, I think we want to avoid creating a nightmare of a debugging problem for some future test cluster (ours or somebody else's) that does not conform to our current specific configuration.

So rather than confining to a list of servers can we make the list all excluding the server (type?)s that you don't want to include?

Reply:
Unfortunately, we are already in nightmare debugging territory with these tests. 

To be clear, my goal here is to get these tests running with some semblance of reliability; it is not to carry this work forward any further than it needs to. There are a number of abstractions we should be making in totality and they go far beyond the scope of this work.

For instance, the config should not operated on directly in the general case. Instead there should be functions that interface with it in a *read only* fashion (we currently mutate the config during our first test).